kyaml/yaml/rnode_test.go

```diff
@@ -1029,26 +1029,19 @@ func TestRNodeMatchesAnnotationSelector(t *testing.T) {
 		"select_01": {
 			selector: ".*",
 			matches:  false,
-			errMsg:   "name part must consist of alphanumeric character",
+			errMsg:   "invalid selector:",
 		},
 		"select_02": {
-			selector: "area=51",
-			matches:  true,
-		},
-		"select_03": {
-			selector: "area=florida",
+			selector: "invalid/annotation/name=fail",
 			matches:  false,
+			errMsg:   "Invalid value:",
 		},
-		"select_04": {
-			selector: "area in (disneyland, 51, iowa)",
+		"select_03": {
+			selector: "area=51",
```

Comment:
Please add another test for 

 "area=51",greeting= "Take me to your leader."

and then just the latter (greeting, no area)